commands.py

```diff
@@ -11,6 +14,8 @@
 ERROR_DATE_NOT_VALID           = "Oops! **%02d/%02d/%04d** is not a valid date in the **future**!"
 ERROR_INVALID_COMMAND          = "`rsvp set %s` is not a valid RSVPBot command! Type `rsvp help` for the correct syntax."
 ERROR_LIMIT_REACHED            = "Oh no! The **limit** for this event has been reached!"
+ERROR_MISSING_MOVE_DESTINATION = "`rsvp move` requires a Zulip stream URL destination (e.g. 'https://zulip.com/#narrow/stream/announce/topic/All.20Hands.20Meeting')"
```

Comment:
I really don't know what to put instead - just replace it with 'Type `rsvp help` for more correct syntax" ?